### ISSUE TITLE
error in the training step

### ISSUE THREAD

**Issue by FabianLeonP** [eiriniar/gleason_CNN#1]
helo! when I try to train the network it's gets 99.50% of accuracy and all classes are gleason 3.

**Comment by eiriniar** [comment]
Hello, there was indeed a bug in the data augmentation script that effectively disabled it. As a result the network would immediately overfit. This has been fixed now.
